test/MUXControlsTestApp/ThemeResourcesTests.cs

```diff
@@ -122,7 +122,8 @@ public void VerifyOverrides()
             IdleSynchronizer.TryWait();
         }
 
-        [TestMethod]
+        // TO DO [Task 30818605]: Disabling to replace with reflection-based tests in release test app
```

Comment:
We can discuss it in Task 30818605. Since here it's just a merging PR from feature branch to master branch, I will not do big change except merging conflicts

Reply:
OK, makes sense.